_pages/support.md

```diff
@@ -474,3 +476,89 @@ https://idp.int.identitysandbox.gov/openid_connect/authorize?
     </div>
   </div>
 </div>
+
+## Contacting Partner Support
+
+**The best way to reach out to the integration experience team is [via our Zendesk portal](https://zendesk.login.gov/)**. We understand email may be more convenient, but it is important that attachments and other information about your issue stay together.  Attachments and email threads can become confusing when forwarded in a Zendesk ticket. For these reasons, we encourage partners to use our [Zendesk portal](https://zendesk.login.gov/) rather than other methods whenever possible. 
+
+If you prefer using Slack, we have a slack channel for partners, **#login-partner-support**. You can request access to this channel by opening a Zendesk ticket. This channel includes other partners and Login.gov engineers that occasionally chime in. It should not be used as a substitute to Zendesk.
+
+<div class="usa-accordion usa-accordion--bordered">
+
+  <h3 class="usa-accordion__heading">
+    <button class="usa-accordion__button" aria-controls="zendesk-guidelines">
+     Zendesk Guidelines
+    </button>
+  </h3>
+  <div id="zendesk-guidelines" class="usa-accordion__container">
+<div class="usa-accordion__content" markdown="1">
+Before submitting your Zendesk ticket, please review the developer documentation on our website. Often the answers we will provide can be found there. 
+When submitting a Zendesk ticket, particularly in cases when you are encountering an error, you can minimize your time to resolve the issue by providing the following information:
+
+ * Link to your dashboard configuration
+ * Screenshots of all error messages
+ * What browser you were using
+ * Time the error(s) occurred or if it is ongoing
+ * Any relevant information recorded in your server logs
+
+It is also **highly recommended** to include someone we can contact from your technical team when creating the Zendesk ticket in case we have further questions about how your application was configured. 
+</div>
+  </div>
+
+
+  <h3 class="usa-accordion__heading">
+    <button class="usa-accordion__button" aria-controls="issues-we-can-help-with">
+      Issues We Can Help With
+    </button>
+  </h3>
+  <div id="issues-we-can-help-with" class="usa-accordion__container">
+<div class="usa-accordion__content" markdown="1">
+Here is a list of common issues partners encounter that our team can assist with. This isn’t a comprehensive list but questions that fall into these categories can usually be answered. 
+
+* Authentication failures while testing in sandbox environment
+* Questions about OIDC/SAML protocols
+* Problems with dashboard configuration (configuration not updating as expected)
+* Problems updating or deploying a production configuration
+* Clarification about something in our developer docs
+* Any errors being returned by Login.gov during authentication
+* Suspected fraudulent activity / RISC notifications 
+* Troubles with redirecting back to your application / SP
```

Comment:
[nitpick] `Service Provider` instead of SP

Reply:
Changed.